src/platform/unix/mod.rs

```diff
@@ -72,6 +68,8 @@ lazy_static! {
     };
 }
 
+static GLOBAL_SHM_COUNT: AtomicUsize = ATOMIC_USIZE_INIT;
```

Comment:
Added a comment and removed `GLOBAL_`. I left the `SHM_` prefix and  named the `lazy_static` pid `SHM_PID`. I don't really like the `SHM_` prefix, but I wanted to make it a little clearer what it was used for.